tests/unit/src/unit/UnitBuilder.hx

```diff
@@ -127,7 +127,7 @@ class UnitBuilder {
 	static public function read(path:String) {
 		var p = Context.makePosition( { min:0, max:0, file:path } );
 		var file = sys.io.File.getContent(path);
-		var code = Context.parseInlineString("{" + file + "}", p);
+		var code = Context.parseInlineString("{" + file + "\n}", p);
```

Comment:
Why?

Reply:
Because if `Something.unit.hx` ends with a line comment then closing curly becomes commented, and `parseInlineString` fails with "Unexpected end of file"